src/spectral.jl

```diff
@@ -264,3 +264,22 @@ function contract(nbt::Nonbacktracking, edgespace::Vector)
     contract!(y,nbt,edgespace)
     return y
 end
+
+"""spectral_distance(G₁, G₂ [, k])
+Compute the spectral distance between graphs G₁ and G₂
+using the top k greatest eigenvalues. If k is ommitted,
+uses k = min(|G₁|,|G₂|).
+
+For further details, please refer to:
+
+JOVANOVIC, I.; STANIC, Z., 2014. Spectral Distances of
+Graphs Based on their Different Matrix Representations
+"""
+function spectral_distance(G₁::SimpleGraph, G₂::SimpleGraph, k::Integer)
+  λ₁ = adjacency_spectrum(G₁)[end-k+1:end]
```

Comment:
Hi @jpfairbanks , is it fine to sort the output of eigs? There is also a warning printed by eigs while adjusting the k. For instance:

```julia
g = random_regular_graph(4,2)
sort(eigs(adjacency_matrix(g), nev=4)[1], rev=true)
```
>WARNING: Adjusting nev from 4 to 3
3-element Array{Float64,1}:
  2.0        
 -3.35344e-16
 -2.0 

What should I do?